serverless/src/retrieveGranuleLinks/fetchCmrLinks.js

```diff
@@ -62,7 +69,7 @@ export const fetchCmrLinks = async ({
   }
 
   const { granules } = granulesData
-  const { cursor: responseCursor, items } = granules
+  const { cursor: responseCursor, items, count } = granules
```

Comment:
I don't really understand why we need the `count` here?

Reply:
yea left over, will remove